source/unified-test-format/schema-1.23.json

```diff
@@ -0,0 +1,1133 @@
+          "type": "string"
+        },
+        "csfle": {
+          "type": "boolean"
+        }
+      }
+    },
+    "entity": {
+      "type": "object",
+      "additionalProperties": false,
+      "maxProperties": 1,
+      "minProperties": 1,
+      "properties": {
+        "client": {
+          "type": "object",
+          "additionalProperties": false,
+          "required": [
+            "id"
+          ],
+          "properties": {
+            "id": {
+              "type": "string"
+            },
+            "uriOptions": {
+              "type": "object"
+            },
+            "useMultipleMongoses": {
+              "type": "boolean"
+            },
+            "observeEvents": {
+              "type": "array",
+              "minItems": 1,
+              "items": {
+                "type": "string",
+                "enum": [
+                  "commandStartedEvent",
+                  "commandSucceededEvent",
+                  "commandFailedEvent",
+                  "poolCreatedEvent",
+                  "poolReadyEvent",
+                  "poolClearedEvent",
+                  "poolClosedEvent",
+                  "connectionCreatedEvent",
+                  "connectionReadyEvent",
+                  "connectionClosedEvent",
+                  "connectionCheckOutStartedEvent",
+                  "connectionCheckOutFailedEvent",
+                  "connectionCheckedOutEvent",
+                  "connectionCheckedInEvent",
+                  "serverDescriptionChangedEvent",
+                  "topologyDescriptionChangedEvent"
+                ]
+              }
+            },
+            "ignoreCommandMonitoringEvents": {
+              "type": "array",
+              "minItems": 1,
+              "items": {
+                "type": "string"
+              }
+            },
+            "storeEventsAsEntities": {
+              "type": "array",
+              "minItems": 1,
+              "items": {
+                "$ref": "#/definitions/storeEventsAsEntity"
+              }
+            },
+            "observeLogMessages": {
+              "type": "object",
+              "minProperties": 1,
+              "additionalProperties": false,
+              "properties": {
+                "command": {
+                  "$ref": "#/definitions/logSeverityLevel"
+                },
+                "topology": {
+                  "$ref": "#/definitions/logSeverityLevel"
+                },
+                "serverSelection": {
+                  "$ref": "#/definitions/logSeverityLevel"
+                },
+                "connection": {
+                  "$ref": "#/definitions/logSeverityLevel"
+                }
+              }
+            },
+            "serverApi": {
+              "$ref": "#/definitions/serverApi"
+            },
+            "observeSensitiveCommands": {
+              "type": "boolean"
+            },
+            "autoEncryptOpts": {
+              "keyVaultNamespace": {
```

Comment:
Update https://github.com/mongodb/specifications/blob/master/source/unified-test-format/tests/Makefile#L1 to use 1.23. That Makefile can be used to validate all files match the schema.

After updating the Makefile, that can be used to identify and fix issues:

```
cd source/unified-test-format/tests
make
```
Results in:
```
error: strict mode: unknown keyword: "keyVaultNamespace"
```